charts/thoras/templates/collector/deployment.yaml

```diff
@@ -98,7 +97,6 @@ spec:
           runAsUser: 1000
           runAsGroup: 0
         {{- end }}
-      {{- end }}
```

Comment:
[boulder] There's another check against `.Values.metricsCollector.timescale.enabled` on line 122 that needs to be removed.

Reply:
Thanks for checking this out! I'll push these changes.

Reply:
+1 Mr eagle eye over here 👁️ lol, I would have totally missed that!